### ISSUE TITLE
Gourmonger Feedback

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#30688]
Having finally bothered getting around to test this engine, I have some feedback.

* This was billed as a quick setup, high maintenance engine, basically an alternate AME. Unfortunately, it's not really quick setup. You really need to have a stockpile of food before you get started, and because it's a radcollector engine, it's reliant on plasma cooling which is half the setup of any other engine. Even a modestly sophisticated gourmonger engine setup will require a chute construction and moving around emitters for easier culling without radding yourself, which is more setup than a singularity.
* The output isn't exciting enough to work as a long setup, high reward engine. If it were the equivalent of a RUST, putting out assloads of power that might make it fun to make a novelty. But your hard work and constant exposure to dangerous radiation levels (demanding you work in a radsuit instead of hardsuit or frequently clear your filter) will not go rewarded. Even with cooled plasma, the output is pitiful and frequently <1MW.
* There's no way to put it on pause. Every other engine has this. Stop the RUST reactions, shut off emitters and space or box the shard, turn off the particle accelerator. If you need to go do something important like fix meteor damage, you can't just pause your maintenance. You have to kill all your gourmongers. Even slimes can be reset from grey extracts if you have to genocide them all, but that's not available here. You can't pack them up into cubes or anything like that. Having some way to pack up your gourmongers really should be added ASAP.

As they are right now, Gourmongers can only really fit as an antagonist tool. Too much commitment and too little reward.

**Comment by Servalfang** [comment]
holy based.
If you want a pause button, you could add a gun or chemical that induces gourmonger hibernation. Jump in and shoot em or throw a gas grenade, and they're essentially on pause on a timer? until reawoken by something? either or really

**Comment by GreatBigFailure** [comment]
I can't disagree. As an engine they're still very flawed. After all the tweaks I think the majority of those flaws are in how our food system/list wasn't designed around balancing an engine like this, and that there's no reliable way to automate them. The latter I can fix by just adding something like a hopper with signaler support, while the latter is much more complicated and something people have been trying to fix (chef vs cargo vs vending machines) for ages. 
With all that said the engine itself does still have flaws I need to work on. To address your feedback:

- I know this ~sounds like~ is backpedaling but the quick setup thing was half a retort to people screaming cargo pizza at me while I was developing the mechanics. I did intend for you to be able to grab a box or two of donk pockets and have it be a functioning engine until your pizza arrived, which is technically possible, but it's also high risk like you said. The quick setup concept became less and less a core part of the design as I had to make more and more compromises due to how our food works. With that said, I think some of the setup you listed is unnecessary. Cooling plasma is only required if you're going for big numbers rather than keeping the lights on and moving emitters is only necessary if you pile food into their pen. The last major tweak to them made splitting much more manageable. Personally I like that they can function as both a low-end quick(ish) and dirty engine and a high-potential creative setup one. Still, nothing you said was wrong.

- I think the output does at the very least fill a niche. Its output scales essentially linearly with no upper limit. The one thing I refused to do the entire time I was making it was tie its metabolism to how much nutrition it had, because that would essentially add a soft-cap to its output. Making the lower ranges of output more usable would certainly help though, maybe giving it a minimum amount of rads released or a base amount added to the scaling amount. 

- Being able to put it back in its cube is a good idea. I had considered this problem fixed for a while because you can, technically, kill it then buy another cube. You lose no nutrition from doing this assuming you feed the new gourmonger the old one's meat. It's a pretty round-about method though. Something like food laced with lipozine cubing it, or a microwave recipe with gourmonger meat to produce a cube might be good.

**Comment by DamianX** [comment]
It's silly and doesn't fit thematically in addition to its design being flawed, as detailed above. I wish to see it gone though I realize this will not happen.

**Comment by GreatBigFailure** [comment]
I disagree that they don't fit thematically. 
"Hey we made this invention that consumes our garbage and produces valuable thing. It even self replicates! Oh no it ate everything!" is a pretty common sci-fi concept. 

desc = "A bio-engineered lifeform designed to recycle excess organic matter into fuel."
Was originally a lot longer but I don't think anyone cares about my original the character

**Comment by Eneocho** [comment]
What if:
Connector plates on a room, gourmongers have hyper efficient bioelectric organs that produce high voltage electric discharges which get absorbed via the connector plates. That would make it fairly more unique and not reliant on blasma.
A side idea to this could be the voltage depending on the quality of the nutrition chem and the amount currently inside the gour. So 10u of Corn Oil produces more power than 10u Nutriment but 20u nutriment produces 2.4 times the power that 10u does. If you want peak power you keep them on the verge of splitting, say, 90 outta 100 nutriment. You can keep them there for stable-ish max power or feed them some extra nutriment for them to go above 100, so they split.

**Comment by Killette2** [comment]
Sure you can pause your gourmonger engine. Just drag them into an emagged trash compactor to recube them, reasonable right?

**Comment by SECBATON-GRIFFON** [comment]
i think monkey recyclers working for more mobs fixes the last point now

**Comment by Eneocho** [comment]
Not really, man. It's not easily accessible, unless you steal the xenobio one, and even then, if the machine behaves as it does with the monkeys, it'd take several gours to get a single gour cube back.

**Comment by SECBATON-GRIFFON** [comment]
> Not really, man. It's not easily accessible, unless you steal the xenobio one,

can't R&D make extra ones?

> and even then, if the machine behaves as it does with the monkeys, it'd take several gours to get a single gour cube back.

i could just make an exemption for these things.
